tests/core/test_optimizers_schedulers.py

```diff
@@ -720,6 +720,7 @@ def test_InverseSquareRootAnnealing(self):
 
     @pytest.mark.unit
     @pytest.mark.run_only_on('CPU')
+    @pytest.mark.pleasefixme
```

Comment:
This test is skipped?

Reply:
Will likely work again after https://github.com/PyTorchLightning/pytorch-lightning/pull/6208 is merged